README.md

```diff
@@ -3,11 +3,41 @@
 Quilt Mappings is a set of open, unencumbered Minecraft mappings, free for everyone to use under the Creative Commons Zero license. The intention is to let 
 everyone mod Minecraft freely and openly, while also being able to innovate and process the mappings as they see fit.
 
+Unlike previous mapping sets, Quilt Mappings allows referencing the official Mojang mappings, which speeds up creating new mappings for Minecraft and improves name quality and accuracy.
+
 To see the current version being targeted, check the branch name!
 
 ## Usage
-To use Quilt Mappings-deobfuscated Minecraft for Minecraft modding or as a dependency in a Java project, you can use [quilt-gradle](https://github.com/quiltmc/quilt-gradle) Gradle plugin. See [fabric wiki tutorial](https://fabricmc.net/wiki/tutorial:setup) for more information.
-
+### A Minecraft mod using Loom
+To use Quilt Mappings in a Fabric or Quilt mod, use the official [Quilt Mappings on Loom](https://github.com/QuiltMC/quilt-mappings-on-loom) Gradle plugin.
+
+`settings.gradle`:
+```groovy
+pluginManagement {
+    repositories {
+        maven { url = "https://maven.quiltmc.org/repository/release" }
+    }
+}
+```
+`build.gradle`:
+```groovy
+plugins {
+  // ...
+  // Check the Quilt Mappings on Loom README for the correct version to use
+  id "org.quiltmc.quilt-mappings-on-loom" version "QMoL_VERSION"
+}
+
+// ...
+
+dependencies {
+   mappings(loom.layered {
+      addLayer(quiltMappings.mappings("org.quiltmc:quilt-mappings:${minecraft_version}+build.${project.quilt_mappings}:v2"))
```

Comment:
```suggestion
      addLayer(quiltMappings.mappings("org.quiltmc:quilt-mappings:${project.minecraft_version}+build.${project.quilt_mappings}:v2"))
```

Also, how does one figure out the quilt mappings version?
Mentioning that might be good.

Reply:
> Also, how does one figure out the quilt mappings version?

That's a great question! What is the best way to determine that?

Reply:
Options:
 - QM Git Tags, a bit hard to order and not something one would think about
 - meta, not user friendly
 - looking at Quilt maven, terrible to navigate
 - my own tool, has some CORS issues with Quilt Maven, not official

Reply:
so there is https://github.com/QuiltMC/quilt-mappings/tags, but thats not the best to look at